test/webauthn/apis.test.js

```diff
@@ -12,27 +12,23 @@
  * License for the specific language governing permissions and limitations
```

Comment:
I'm fine with simple and/or node specific tests here, but other than those I'd prefer if these were moved into the backend-sdk-testing repo. This is fine for now if you think it'd be a lot of overhead because of mocking/using nock, but we'll need to move them.

Reply:
Did it like this because i wanted to have some kind of feedback on the implementation and making sure it works.
We'll move them to the SDK testing repo